### PR TITLE
:bug: gateway fix hazelcast properties

### PR DESCRIPTION
**Pull request by simonhir**
**Description**

Gateway fix missing setter for Hazelcast properties.

```
***************************
APPLICATION FAILED TO START
***************************

Description:

Failed to bind properties under 'refarch.hazelcast' to de.muenchen.refarch.gateway.configuration.HazelcastProperties:

    Property: refarch.hazelcast.service-name
    Value: "refarch-gateway-tmn8mmqgaa"
    Origin: System Environment Property "REFARCH_HAZELCAST_SERVICENAME"
    Reason: java.lang.IllegalStateException: No setter found for property: service-name

Action:

Update your application's configuration
```

### CODE REVIEW

**Review by devtobi — APPROVED** [review]
LGTM

File: shared-files/spotbugs-excludes.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<FindBugsFilter>
+    <Match>
+        <Class name="*"/> <!-- You can specify a specific class name if needed -->
+        <Method name="*"/> <!-- You can specify a specific method name if needed -->
+        <Bug code="EI_EXPOSE_REP2"/>
```
Comment (simonhir) [inline shared-files/spotbugs-excludes.xml:5]: @devtobi we probably should discuss about this.
In my opinion there is no other way, as often it is cleaner that way and it travels up the call tree if suppressed in one spot.

Comment (simonhir) [reply shared-files/spotbugs-excludes.xml]: Reverted and done with Annotation.
Will be discussed for Spotbugs with integrations.

**Review by devtobi — APPROVED** [review]
LGTM

**Pull request merged**